### PR TITLE
Fix: don't use user#authentication_token.length for hidden value

### PR DESCRIPTION
**Pull request by nicolas-brousse**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -35,7 +35,7 @@
               <%= @user.authentication_token %>
             </span>
             <span data-reveal-target="item">
-              <%= "*" * @user.authentication_token.length %>
```
Comment (B-Rass) [inline app/views/users/show.html.erb:38]: Why setting a default value (20) and not `0`?

Comment (nicolas-brousse) [reply app/views/users/show.html.erb:38]: It looked weird. Maybe the better should be to simple show empty value and hidden reveal too

Comment (B-Rass) [reply app/views/users/show.html.erb:38]: yes

Comment (nicolas-brousse) [reply app/views/users/show.html.erb:38]: Updated

**Review by B-Rass — APPROVED** [review]

**Pull request merged**